charts/actions-runner-controller/templates/deployment.yaml

```diff
@@ -67,6 +67,9 @@ spec:
         {{- if .Values.runner.statusUpdateHook.enabled }}
         - "--runner-status-update-hook"
         {{- end }}
+        {{- if .Values.enableProdLogConfig  }}  
+        - "--enable-prod-log-config={{ .Values.enableProdLogConfig }}"
```

Comment:
Hey! Sorry for the delayed review. Can we make this look more like a standard flag, like `--log-format=[text|json]`? `json` will be the value specified when you'd need structural logging.

Reply:
Np, updated following your comments